src/routes/news/components/TunewsDetailsFooter.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+
+import React from 'react'
+import styled from 'styled-components'
+import { TFunction } from 'i18next'
+import type Moment from 'moment'
+
+const Footer = styled.footer`
+  display: flex;
+  width: 96%;
```

Comment:
This width seems a little bit weird. Was this applied in order to fix some weird behavior?

Reply:
Yeah. I changed that.